questions.md

```diff
@@ -0,0 +1,2 @@
+# What is PR?
+# Is it associated with Issues anyway?
```

Comment:
A pull request is a type of issue. Here are the docs on working with those: https://help.github.com/en/categories/collaborating-with-issues-and-pull-requests